widget/tabcontainer.go

```diff
@@ -79,7 +79,7 @@ func (t *TabContainer) CurrentTab() *TabItem {
 
 // SelectTabIndex sets the TabItem at the specific index to be selected and its content visible.
 func (t *TabContainer) SelectTabIndex(index int) {
-	if index < 0 || index >= len(t.Items) || t.current == index {
```

Comment:
Why are no not ignoring tapping the current tab any more?

Reply:
2 things :

- its not just taps that SelectTabIndex
- without this change, get a situation where the 1st tab in a container does not correctly set the text color, because this line skips the refresh below

You dont notice it when the PrimaryTextColor and TextColor are the same value